tools/c7n_sphinxext/c7n_sphinxext/_templates/schema.rst

```diff
@@ -1,9 +1,12 @@
 
 .. container:: toggle
 
-  .. container:: header
+  ..
+    `raw:: html` is used because `.. container` doesn't support empty content
```

Comment:
can we do this as a jinja comment so its not in the omitted output. http://jinja.pocoo.org/docs/2.10/templates/#comments